qiskit_ibm_provider/ibm_backend.py

```diff
@@ -309,7 +309,7 @@ def dtm(self) -> float:
         Returns:
             dtm: The output signal timestep in seconds.
         """
-        return self._configuration.dtm
+        return self._configuration.dtm * 1e-9
```

Comment:
I am confused as to why we need this. `self._configuration` is an instance of `QasmBackendConfiguration` or `PulseBackendConfiguration` and they already do the conversion from nanosecs to secs.

Reply:
Yeah, this was my mistake. I just ported what we did for the fake backends in terra but they don't use the classes and I forgot the provider package did.  I'll drop this we only need to make the `Delay` change here.

Reply:
```suggestion
        return self._configuration.dtm
```